src/MyHealth/DayIndicator.tsx

```diff
@@ -0,0 +1,145 @@
+import React, { FunctionComponent } from "react"
+import { View, Text, ViewStyle, TextStyle, StyleSheet } from "react-native"
+import dayjs from "dayjs"
+
+import { DateTimeUtils } from "../utils"
+import { Outlines, Colors, Typography, Spacing } from "../styles"
+import { CheckInStatus, DayLogData } from "./symptoms"
+
+interface DayIndicatorProps {
+  logData: DayLogData
+  isSelected: boolean
+}
+
+type IndicatorStyle = [ViewStyle, TextStyle]
+
+const DayIndicator: FunctionComponent<DayIndicatorProps> = ({
+  logData,
+  isSelected,
+}: DayIndicatorProps) => {
+  const isToday = DateTimeUtils.isToday(logData.date)
+
+  const applyRiskStyle = ([
+    circleStyle,
+    textStyle,
+  ]: IndicatorStyle): IndicatorStyle => {
+    switch (logData.checkIn.status) {
+      case CheckInStatus.NotCheckedIn: {
+        return [{ ...circleStyle }, { ...textStyle, color: Colors.primaryText }]
+      }
+      case CheckInStatus.TooOld: {
```

Comment:
This CheckInStatus.TooOld seems to be a bit out of place. Was the reason it was included to be able to have a calendar view display a different color for days older that 14? If thats the case then it feels like we are adding complexity to the data model to accommodate a the view layer. What do you think about keeping the data model agnostic of the UI solution by having a view layer only solution to encode this?  something like
```
type CalendarHistory = CalendarDatum[]
interface CalendarDatum = {
   date: Posix
   logData: LogData | null
}
```
then this applied style would be something like:
```
  if (logData.checkIn) {
    return hasDataStyle
  } else {
     return noDataStyle
  }
```
This way if we ever change up the calendar or the data model, the view logic wont be coupled to the data model.